source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,101 @@
+.. _csharp-update-many:
+
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can update more than one document at once using the `UpdateMany() <{+api-root+}/M_MongoDB_Driver_IMongoCollection_1_UpdateMany.htm>`__ method on
+a ``MongoCollection`` object. This method requires a **query filter**, which specifies which documents to update, and an **update** statement, which specifies the changes the driver should make to all documents matching the query filter.
+
+.. tip::
+
+   You can pass an instance of `UpdateOptions <{+api-root+}/T_MongoDB_Driver_UpdateOptions.htm>`__ to the ``UpdateMany()`` method in
+   order to customize its behavior.
+
+Examples
+--------
+
+These examples use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: ../includes/code-examples/update-many/Restaurant.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language: csharp
+   :copyable:
+   :dedent:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: update-many-async
+
+      Update Many Documents Asynchronously
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Issue: The titles on the right hand nav do not take you to the expected example.

Suggestion: Remove the titles from the tabs. I think it makes sense to discuss this as 1 example in an async and sync state (I.E change the title from "Examples" to "Example" and then introduce the example outside of the tabs. This is how it is done for [Node](https://www.mongodb.com/docs/drivers/node/current/usage-examples/findOne/) (JS/TS) for example).

Reply:
I'm not sure I understand. Present just the async code (as the Node page does)?

Incidentally, shouldn't the Node page have an "On This Page" widget? When it does, won't the JavaScript/TypeScript tabs be broken?

Reply:
We discussed this sync and clarified initial comment. Seemed like we were alright with initial suggestion. Don't see the edit yet, so I'll leave this thread open.